ra-tls/README.md

```diff
@@ -12,14 +12,26 @@ cd $src/ra-tls
 make
 ```
 
-# Run
+# Run with SGX
 ``` shell
 mkdir -p /run/rune
 cd build/bin
-./ra-tls-server run &
+./ra-tls-server run --epid &
+
+# Run client in SGX
+./elv echo --epid
```

Comment:
I hope we have a common option called --attestation / -a, and the allowed attestation mode should include epid or ecdsa.